src/Controller/EnMarche/ReferentController.php

```diff
@@ -2,19 +2,12 @@
 
 namespace App\Controller\EnMarche;
 
-use App\Address\GeoCoder;
 use App\Entity\Adherent;
-use App\Entity\InstitutionalEvent;
 use App\Entity\ReferentOrganizationalChart\PersonOrganizationalChartItem;
-use App\Form\InstitutionalEventCommandType;
 use App\Form\ReferentPersonLinkType;
-use App\InstitutionalEvent\InstitutionalEventCommand;
-use App\InstitutionalEvent\InstitutionalEventCommandHandler;
 use App\Intl\FranceCitiesBundle;
-use App\Referent\ManagedInstitutionalEventsExporter;
```

Comment:
could we remove the file `ManagedInstitutionalEventsExporter.php`?

Reply:
indeed, it seems not used anymore, removed